.gitignore

```diff
@@ -12,6 +12,9 @@ build/
 Gemfile.lock
 vendor/
 
+# temp directory is ignored so that it can be used to store temporary files such as local testing configurations.
+temp/
```

Comment:
I found that I also have this folder inside AZ folder. Do you know which azkaban module generate this folder?